WcaOnRails/Gemfile.lock

```diff
@@ -738,4 +744,4 @@ DEPENDENCIES
   wicked_pdf
 
 BUNDLED WITH
-   2.1.4
+   2.2.24
```

Comment:
Please either
- bundle with 2.1.4, so that this version number doesn't constantly change with every PR that members with different local `Bundler` versions end up creating
- or share your Ruby wisdom on how to avoid/suppress this line altogether, it has been causing us trouble with onboarding in the past (people had to deliberately install bundler 2.1.4 instead of "just" the latest bundler)

Reply:
Ah good point. I've installed bundler 2.1.4 to get rid of this part of the diff.

I unfortunately don't know what the "ruby way" is of handling this sort of thing. It is super annoying how easy it is for it to change when you have a bunch of people all working on the same repo. Is bundler supposed to become a core part of ruby someday...?